client/app/bundles/course/survey/pages/ResponseIndex/index.jsx

```diff
@@ -220,23 +219,23 @@ class ResponseIndex extends React.Component {
                 <FormattedMessage {...surveyTranslations.opensAt} />
               </TableRowColumn>
               <TableRowColumn>
-                {formatDateTime(survey.start_at)}
+                { survey.start_at ? moment(survey.start_at).format(longDateTime) : '' }
               </TableRowColumn>
             </TableRow>
             <TableRow>
               <TableRowColumn>
                 <FormattedMessage {...surveyTranslations.expiresAt} />
               </TableRowColumn>
               <TableRowColumn>
-                {formatDateTime(survey.end_at)}
+                { survey.end_at ? moment(survey.end_at).format(longDateTime) : '' }
```

Comment:
Have updated with helper functions to format longDateTimes and shortDateTimes, since they are the most common!